components/script/dom/globalscope.rs

```diff
@@ -3266,7 +3266,7 @@ impl GlobalScope {
                         let navigator = window.Navigator();
                         if let Some(gamepad) = navigator.get_gamepad(index) {
                             if window.Document().is_fully_active() {
-                                gamepad.update_connected(false, gamepad.exposed());
+                                gamepad.update_connected(false, gamepad.exposed(), can_gc);
```

Comment:
Since this is only used within a task! macro, we don't actually want to.propagarethis as an argument to handle_gamepad_disconnect.